src/f0cal/bootstrap/saltbox/srv/salt/f0cal/bootstrap/user/init.sls

```diff
@@ -11,9 +10,20 @@
 {% set pip_exe = "pip" %}
 
 {% if venv_dir %}
+venv_bin:
+  file.managed:
+    - makedirs: True
+    - name: {{ temp_dir }}/.pyvenv
+    - mode: 777
+    - contents: |
+        #! /bin/bash
+        {{ python_exe}} -m venv $@ && \
+        $1/bin/pip install --upgrade pip
```

Comment:
@AndreyShprengel I'm mixed on this line for cases where we don't absolutely need it. What do you think?

Reply:
Just because it takes extra time? I suppose we could do it further down the line but I think when using constraints file its safer to have more recent pip.

Reply:
Removed